### PR TITLE
Show "manual flush" on Display with Temperature

### PR DESCRIPTION
**Pull request by genkigenki**
Show "manual flush" on Display with Temperature

### CODE REVIEW

**Comment by fiendie** [comment]
You committed a backup of your userConfig.h
Also, if you're already rebasing, please squash the "Rebase" commit into the previous one.

**Comment by genkigenki** [comment]
> You committed a backup of your userConfig.h Also, if you're already rebasing, please squash the "Rebase" commit into the previous one.

Done.

**Comment by genkigenki** [comment]
Changed:
- Showing brewtime (hint from @LoQue90 - thanks)
- Moved duplicated code to show brew time into function
- Analogous to the "°" for temperature, added small "s" for the brewtime as physical unit (as we also have weight now)

**Comment by LoQue90** [comment]
rebase just works, nothing to do here.
I just moved some lines, just style cleanup.
removed "and rebase" from the commit
if you want just pull: https://github.com/LoQue90/clevercoffee/tree/manuall_flush

File: src/display.h [hunk 1]
```diff
@@ -121,6 +121,52 @@ void drawTemperaturebar(int x, int y, int heightRange) {
     }
 }
 
+/**
+ * @brief Draw the temperature in big font at given position
+ */
+void displayTemperature(int x, int y) {
+
```
Comment (LoQue90) [inline src/display.h:128]: remove empty line

File: src/display.h [hunk 2]
```diff
@@ -121,6 +121,52 @@ void drawTemperaturebar(int x, int y, int heightRange) {
...
+    u8g2.setFont(u8g2_font_profont15_tf);
+
+    if (brewtime < 10000.000) {
+        u8g2.setCursor(x+67, y+14);
+    }
+    else {
+        u8g2.setCursor(x+69, y+14);
+    }
+    u8g2.print("s");
```
Comment (LoQue90) [inline src/display.h:166]: add empty line before

File: src/display.h [hunk 3]
```diff
@@ -121,6 +121,52 @@ void drawTemperaturebar(int x, int y, int heightRange) {
...
+
+    if (brewtime < 10000.000) {
+        u8g2.setCursor(x+16, y);
+    }
+    else {
+        u8g2.setCursor(x, y);
+    }
+
+    u8g2.print(brewtime / 1000, 1);
+
```
Comment (LoQue90) [inline src/display.h:157]: remove empty line

**Review by LoQue90 — CHANGES_REQUESTED** [review]

**Pull request merged**